### DISCUSSION TITLE
Is there a reason why @react-spectrum/pagination is not publicly available?

### DISCUSSION THREAD

**Discussion by dzmitry-kankalovich** [adobe/react-spectrum#1712] [Q&A] [answered]
Hi,

We've noticed there is a [pagination](https://github.com/adobe/react-spectrum/tree/main/packages/@react-spectrum/pagination) component, which is not publicly available over npm.

I've checked the storybook and git history - it appears to be ready for usage. Is there any particular reason why it is still in private?

Regards,
Dmitry

**Comment by snowystinger** [comment, marked as answer]
This component was updated in Design and hasn't made its way back to our priorities yet.
I don't know how much testing has been done on it, or on the hooks themselves, so there's probably work that still needs to be done there as well.
With the changes to the design, we probably also need to make sure our APIs still makes sense.
As such, we aren't ready to release what we currently have.

**Reply by dzmitry-kankalovich** [reply]
Thanks Robert,

Appreciate response.

Regards,
Dima

**Reply by steveoh** [reply]
I was converting a bootstrap pager today and was a little surprised there was nothing available from react-aria. Since it's now 4 years later, are there any updates or new thoughts on paging? It seems like a staple component with tricky aria needs.

**Reply by snowystinger** [reply]
We've not had any priority for it, so no updates right now. We're focused on the update to Spectrum and all associated RAC/hook updates for those components. We'll probably revisit after that work is done.

**Reply by steveoh** [reply]
Can I read about those updates somewhere? Is it related to react 19 or something else?

**Reply by snowystinger** [reply]
It's the implementation of https://s2.spectrum.adobe.com/
All of our updates are here: https://react-spectrum.adobe.com/releases/index.html I don't have anything special beyond that.
